Libplanet/Address.cs

```diff
@@ -96,6 +96,28 @@ public Address(PublicKey publicKey)
         {
         }
 
+        /// <summary>
+        /// Derives the corresponding <see cref="Address"/> from a hexadecimal
+        /// address string.
+        /// </summary>
+        /// <exception cref="ArgumentNullException">Thrown when <c>null</c> was
+        /// passed to <paramref name="hex"/>.</exception>
+        /// <exception cref="ArgumentException">Thrown when the given <paramref
+        /// name="hex"/> did not lengthen 40 bytes.</exception>
```

Comment:
It's technically not bytes but characters.

Reply:
You're right. I'll fix it.